packages/teleport/src/components/BannerList/Banner.test.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { render, fireEvent, screen } from 'design/utils/testing';
+
+import { Banner } from './Banner';
+
+describe('components/BannerList/Banner', () => {
+  it('displays the supplied message', () => {
+    const msg = 'I am a banner';
+    render(
+      <Banner message={msg} severity={0} id="test-banner" onClose={() => {}} />
+    );
+    expect(screen.getByText(msg)).toBeInTheDocument();
+  });
+
+  function assertComputedStyle(
```

Comment:
Have you tried to use https://github.com/styled-components/jest-styled-components#tohavestylerule?

Reply:
I did, and it mostly works but requires another dependency and you can see a bit more detailed description in the email.

Reply:
> Styled Components has a [tohavestylerule](https://styled-components.com/docs/tooling#tohavestylerule) assertion for Jest for this purpose but I found it didn't work if you had any wrappers, like a `<Router>`.

Can you provide an example? I wasn't able to reproduce this issue

Reply:
I switched to using the `toHaveStyleRule` as the aforementioned issue wasn't actually in this pr, just something from my testing with it.